src/helm/helm.ts

```diff
@@ -26,8 +26,13 @@ export type HelmRelease = {
  * @returns a list of all Helm releases in the current namespace on the current cluster
  */
 export async function getHelmReleases(): Promise<HelmRelease[]> {
-    const res = await CliChannel.getInstance().executeTool(HelmCommands.listHelmReleases(), undefined, false);
-    return JSON.parse(res.stdout) as HelmRelease[];
+    try {
+        const res = await CliChannel.getInstance().executeTool(HelmCommands.listHelmReleases(), undefined, false);
+        return JSON.parse(res.stdout) as HelmRelease[];
+    } catch (err) {
+        window.showErrorMessage(`Cannot list HELM Releases: ${err}`);
```

Comment:
IMHO it makes more sense to let the error be passed on to whatever method is calling this one. What do you think?

Reply:
When this method (and yet another one from the `osWrapper` with try-catch block added) are invoked from `OpenShiftExplorer.getChildren()` it breaks the method execution if `JSON.parse()` fails to process the `stdout` of the result - this makes the explorer tree fail on update. I could actually ignore the error (as it's usually "json is malformed"-like error, but I'd like to know when it usually happens

Reply:
Okay sounds good!